shared_model/validators/field_validator.hpp

```diff
@@ -48,27 +48,37 @@ namespace shared_model {
           ReasonsGroupType &reason,
           const interface::types::AccountIdType &account_id) const {
         if (not std::regex_match(account_id, account_id_)) {
-          reason.second.push_back("Wrongly formed account_id");
+          auto message =
+              generateErrorMessage("Wrongly formed account_id",
```

Comment:
*couldhave*
As for error message, it would be good to return matching regex to the user in error message, so the user will be able to fix his code without having to go to the documentation.

Reply:
Isn't it insecure to expose regexes?

Reply:
1. Software is opensource, users are able to look at them by themselves
2. It is better for UX

Reply:
Okay, will add.

Reply:
Sadly, there seems to be no way to extract a pattern from a string, so this change would require putting pattern in a separate variable. I would do that in separate pull request along with separation of .hpp and .cpp file.